apps/cow-fi/components/Layout/index.tsx

```diff
@@ -50,22 +51,28 @@ export function Layout({ children, bgColor, host }: Readonly<LayoutProps>) {
     <>
       <GlobalStyles />
       <LocalStyles />
-      <MenuBar
-        navItems={NAV_ITEMS}
-        productVariant={PRODUCT_VARIANT}
-        additionalNavButtons={NAV_ADDITIONAL_BUTTONS}
-        padding="10px 60px"
-        maxWidth={PAGE_MAX_WIDTH}
-        LinkComponent={LinkComponent}
-      />
+      {/* Override global light theme to force dark mode for MenuBar only */}
+      <ThemeProvider theme={baseTheme('dark')}>
```

Comment:
Result of `baseTheme('dark')` should be cached in a cost to prevent excessive re-renders (and bellow as well)

Reply:
Addressed now